addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -0,0 +1,77 @@
+# iCloud Binding
+
+The Apple [iCloud](https://www.apple.com/icloud/) is used to retrieve data such as the battery level or current location of one or multiple Apple devices connected to an iCloud account.
+Updates are quick and accurate without significant battery time impact.
+The Binding also offers access to the "Find My iPhone" function.
+
+An Apple account is required. Two factor authentication is supported.
+
+Please note: Application specific passwords are not supported.
+You´ll need to provide the account password to the binding.
+
+## Discovery
+
+An iCloud account can be added as a thing and needs to be configured with your ID and password.
+The devices registered to this account will then be automatically discovered.
+
+## Binding Configuration
+
+The Binding has no configuration options, all configuration is done at Thing level.
+
+## Thing Configuration
+
+The Binding provides two Thing types.
+
+### iCloud Account (Bridge) Thing
```

Comment:
Imho you can remove the "(Bridge)" here. Doesn't add any value. Instead add it to the introduction as suggested above.